skill-tree /src/main/java/com/RDS/skilltree/EndorsementList/EndorsementListModel.java

```diff
@@ -0,0 +1,49 @@
+package com.RDS.skilltree.EndorsementList;
+
+import com.RDS.skilltree.Endorsement.EndorsementModel;
+import com.RDS.skilltree.User.UserModel;
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Entity
+@NoArgsConstructor
+@Data
+@Table(name = "endorsement_list")
+public class EndorsementListModel {
+    @Id
+    @GeneratedValue
+    @Column(name = "id", columnDefinition = "BINARY(16)")
+    private UUID id;
+
+    @JoinColumn(name = "endorsement_id", referencedColumnName = "id")
+    @ManyToOne(targetEntity = EndorsementModel.class, cascade = CascadeType.ALL)
+    private EndorsementModel endorsement;
+
+    @JoinColumn(name = "user_id", referencedColumnName = "id")
+    @ManyToOne(targetEntity = UserModel.class, cascade = CascadeType.ALL)
+    private UserModel endorser;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "is_deleted", nullable = false)
+    private boolean isDeleted;
+
+    @Column(name = "type", nullable = false)
+    private EndorsementType type;
+
+    @Column(name = "created_at", nullable = false)
```

Comment:
what about created_by, modified_by/_at

Reply:
Wasn't mentioned in the ER diagram

- https://github.com/Real-Dev-Squad/skill-tree-backend/issues/2